tests/mol-navigation/EPUB/ch2.xhtml

```diff
@@ -0,0 +1,11 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta charset="utf-8"/>
+        <title>Chapter 2</title>
+        <link rel="stylesheet" type="text/css" href="css/base.css"/>
+    </head>
+    <body id="body">
+        <h1 id="mo-1">Chapter 2</h1>
+        <p id="mo-2">This page should play when "Chapter 2" is selected from the table of contents.</p>
```

Comment:
For clarity, I'd recommend something explicit like "Test passes if the media overlay audio on this page plays when..." (and does the reader need to enable MO on the first page before doing that? If so, please say so.)

Reply:
The instructions start on chapter 1, please see that page first and then let me know if it still doesn't make sense.

I will update the wording to say "The test passes if..."

Reply:
(Marisa did this.)